service/src/block_device.rs

```diff
@@ -597,6 +563,8 @@ mod tests {
         assert!(mgr.get_config(&key).is_some());
 
         let mgr = Arc::new(mgr);
+        //assert with wrong blob_id
```

Comment:
Should add space between `//` and `assert`.

Reply:
Thanks for your suggestion! I submit a new commit.